shared/ghutil/pullrequest.go

```diff
@@ -133,6 +133,35 @@ func (gc *GithubClient) GetPullRequest(org, repo string, ID int) (*github.PullRe
 	return res, err
 }
 
+// GetPullRequestByCommitID gets PullRequest by commit ID
+func (gc *GithubClient) GetPullRequestByCommitID(org, repo string, commitID string) (*github.PullRequest, error) {
```

Comment:
Your may also want to add it to fake client so it's easier for you writing unit tests later

Reply:
Added.